components/gitpod-db/src/typeorm/migration/1612448906950-ExampleRepositories.ts

```diff
@@ -0,0 +1,36 @@
+import {MigrationInterface, QueryRunner} from "typeorm";
```

Comment:
please, add copyright headers in all new files.

Reply:
Done!